packages/manager/apps/web/client/app/hosting/offer/upgrade/hosting-offer-upgrade.html

```diff
@@ -59,29 +59,20 @@
                 data-description="{{:: 'hosting_order_upgrade_select_duration' | translate }}"
                 data-on-focus="$ctrl.getDurations()"
                 data-loading="$ctrl.loading.durations"
-                data-valid="$ctrl.durations.available"
+                data-valid="$ctrl.durations"
```

Comment:
```suggestion
                data-valid="$ctrl.durations && $ctrl.model.duration"
```
if not your button can be available without to have oui-radio selected